src/components/IconButton/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Icon = styled.img`
+    width: ${props => (props.width ? props.width : null)};
+    height: ${props => (props.height ? props.height : null)};
```

Comment:
I think that it is a good practice to always set default values. So something like:
```
width: ${props => props.width || 40};
height: ${props => (props.height || 40};
```

This way, if someone won't specify width/height - it will take the default value (40 in this case)

Reply:
hmm if width will be null, than there will be no rule for width ? I mean, by setting to null I was hoping that it will set something like "inherit" or something, but probably it's not. I will change it to some default than ;)